crates/sui-indexer/src/schema.rs

```diff
@@ -194,7 +199,7 @@ diesel::table! {
         recipients -> Array<Nullable<Text>>,
         checkpoint_sequence_number -> Int8,
         transaction_time -> Nullable<Timestamp>,
-        transaction_kind -> Nullable<Text>,
+        transaction_kinds -> Array<Nullable<Text>>,
```

Comment:
ditto

Reply:
there was a schema mismatch, Patrick fixed it in a different direction.
Still making the change here from Nullable<Text> to Text b/c Text is the generated type while running `diesel database reset`, had to change a couple of other places to make it build.